sdk/appplatform/Azure.ResourceManager.AppPlatform/src/Customize/Models/AppPlatformServiceProperties.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.AppPlatform.Models
+{
+    public partial class AppPlatformServiceProperties
+    {
+        /// <summary> DEPRECATED: ServiceInstanceEntity GUID which uniquely identifies a created resource. </summary>
```

Comment:
nit: can we add something saying it can throw a FormatException if the value from the service is not a Guid